gatsby-node.js

```diff
@@ -43,4 +51,56 @@ exports.createPages = async ({ actions }) => {
       },
     });
   });
+
+  return graphql(`
+    {
+      allPage {
+        edges {
+          node {
+            id
+            slug
+            title
+            type
+            branding {
+              cover_image
+              banner_border_color
+              texture_image
+              organization_logo {
+                url
+                alt
+              }
+            }
+          }
+        }
+      }
+    }
+  `).then((result) => {
+    let pageType;
+    if (result.data) {
+      result.data.allPage.edges.forEach(({ node }) => {
+        if (node.type in PAGE_TYPES) {
+          pageType = PAGE_TYPES[node.type];
+          createPage({
+            path: node.slug,
+            component: path.resolve(`./src/templates/${pageType.template}`),
```

Comment:
Instead of defining a `templates` folder, can we resolve the path to the `ProgramPage` component itself? That component has `IntlProvider` and `Layout` already.

```jsx
const PAGE_TYPES = {
  'pages.ProgramPage': {
    template: './src/components/masters/program/ProgramPage.jsx', // this will be the path once my refactor PR merges in later today
  },
};

createPage({
  // ...
  component: path.resolve(pageType.template},
});
```